demo.py

```diff
@@ -93,8 +100,8 @@ def train_model(X_train, y_train, X_test, y_test, lmd):
     y_train_pred = clf.predict(X_train)
```

Comment:
I just went ahead and dropped the "Fitting..." and "Testing..." lines, since they're unnecessary and they weren't consistently used for all four methods.